pkg/investigations/ocmagentresponsefailure/ocmagentresponsefailure_test.go

```
@@ -0,0 +1,182 @@
+
+			// Build resources
+			resources := &investigation.Resources{
+				Cluster:   m.cluster,
+				OcmClient: m.ocmClient,
+				PdClient:  m.pdClient,
+				AwsClient: m.awsClient,
+				Notes:     notewriter.New(tt.name, logging.RawLogger),
+			}
+
+			if tt.experimentalEnabled {
+				os.Setenv("CAD_EXPERIMENTAL_ENABLED", "true")
```

golangci-lint (2.11.4)

[error] Error return value of `os.Setenv` is not checked

(errcheck)